docs/explanation/chisel.rst

```diff
@@ -44,3 +44,15 @@ could see as a block of stone from which we can carve and extract only the
 small and relevant parts that we need to run our applications, thus keeping
 ROCKs small and less exposed to vulnerabilities.
 
+Slice definitions
+~~~~~~~~~~~~~~~~~
+
+A package's slices can be defined via a YAML slice definitions file. Check the
+`slice definitions reference
+<https://github.com/canonical/chisel/#slice-definitions>`_ for more information
+about this file's format.
+
+.. note::
```

Comment:
i think only the ``Contributions are welcome and encouraged.`` need to be in note

Reply:
well, a "note" is for making a **remark** about something. I don't think that fits